<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Group</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Brand</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Complete</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Acceptance date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Execution date</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchGroup"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Brand:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchBrandEditLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Acceptance date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="acceptanceDateEditLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Execution date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="executionDateEditLIne"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="unknownRadioButton">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="incompleteRadioButton">
          <property name="text">
           <string>Incomplete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="completeRadioButton">
          <property name="text">
           <string>Complete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
